PhotoBooth/PhotoShootSetupCardView.swift

```diff
@@ -0,0 +1,308 @@
+import UIKit
+
+class PhotoShootSetupCard: UIView {
+
+    let fontRegular = "AvenirNext-Regular"
+    let fontBold = "AvenirNext-Bold"
+
+    let photoBoothBlue = UIColor(red: 38/225, green: 185/225, blue: 224/225, alpha: 1)
+
+    lazy var cardUIView: UIView = {
+        let view = UIView()
+        view.backgroundColor = .white
+        view.layer.opacity = 0.95
+        view.layer.cornerRadius = 20
+        view.layer.masksToBounds = true
+        view.translatesAutoresizingMaskIntoConstraints = false
+        return view
+    }()
+
+    lazy var headerLabel: UILabel = {
+        let label = UILabel()
+        label.text = "Setup your photoshoot"
+        label.font = UIFont(name: fontBold, size: 22)
+        label.textAlignment = .center
+        label.textColor = .black
+        label.backgroundColor = .clear
+        label.numberOfLines = 0
+        label.adjustsFontSizeToFitWidth = true
+        label.translatesAutoresizingMaskIntoConstraints = false
+        return label
+    }()
+
+    lazy var photosStepperContainerUIView: UIView = {
+        let view = UIView()
+        view.translatesAutoresizingMaskIntoConstraints = false
+        return view
+    }()
+
+    lazy var photosStepperMinusButton: UIButton = {
```

Comment:
you should make the stepper its own class. Then you can reuse it else where without duplicate set up. Usually large  duplications of code is a sign that you can abstract things out to a function or another class.